atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/DBMgrConfigurationException.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2015 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.nexus.db.pool;
+
+import com.palantir.exception.PalantirSqlException;
+
+/**
+ * Exception type thrown by the {@link DBMgr} when it is misconfigured.
```

Comment:
yeah, this entire class should be ported back and replaced with an oss-only exception (no reason for atlas and internal code to share this exception type)